wallet/chainntfns.go

```diff
@@ -107,15 +106,32 @@ func (w *Wallet) handleChainNotifications() {
 					chainClient, birthdayStore,
 				)
 				if err != nil && !waddrmgr.IsError(err, waddrmgr.ErrBirthdayBlockNotSet) {
-					panic(fmt.Errorf("unable to sanity "+
-						"check wallet birthday block: %v",
-						err))
```

Comment:
It looks like the wallet is ok with a nil birthday below, so I do not think it is necessary to panic here.